CHAOS-1636: Chaos 1636 relnotes (#1611)
harness/developer-hub: release-notes/chaos-engineering.md

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
:::

> ## Latest - May 5, 2023, version 0.11.x

Reviewer: ```suggestion
## Latest - May 5, 2023, version 0.11.1
```

We take chaos-manager version.

```mdx-code-block
<Tabs>
  <TabItem value="What's new">
```

> * A GameDays page has been added to HCE. (CHAOS-643)

Reviewer: ```suggestion
* Introduction of GameDays in HCE Module. (CHAOS-643)
```

Does it look good?
